src/Hl7.Fhir.STU3.Specification.Tests/Source/ZipSourceTests.cs

```diff
@@ -16,7 +17,8 @@ public class ZipSourceTests
         public void ListSummariesIncludingSubdirectories()
         {
             var zipfile = Path.Combine("TestData", "ResourcesInSubfolder.zip");
-            var zip = new ZipSource(zipfile, new DirectorySourceSettings() { IncludeSubDirectories = true });
+            var extractDir = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
+            var zip = new ZipSource(zipfile, extractDir, new DirectorySourceSettings() { IncludeSubDirectories = true });
```

Comment:
Why should we pass here the extract directory? There is still a constructor of `ZipSource` that generates a default extraction directory. Right?

Reply:
I did half of the unit tests against the constructor with one arg, and the other half against the constructor with two args to be able to test both. But it is a bit explicit (and lazy), I'll create a few more tests.